### PR TITLE
feat(discover): Add docs for organization_events

### PR DESCRIPTION
**Pull request by wmak**
- This adds a new doc sections for Visibility called `Discover and Performance`
- This adds the first Discover endpoint we're publicizing the `events` endpoint (which replaces the soon to be deprecated `eventsv2` endpoint) as the first endpoint in the documentation

### Preview
image

### CODE REVIEW

File: src/sentry/apidocs/parameters.py [hunk 1]
```diff
@@ -17,6 +18,50 @@ class GLOBAL_PARAMS:
         type=str,
         location="path",
     )
+    STATS_PERIOD = OpenApiParameter(
```
Comment (AniketDas-Tekky) [inline src/sentry/apidocs/parameters.py:21]: I think OpenApiParameter maps to URL parameters, not query parameters

Comment (AniketDas-Tekky) [reply src/sentry/apidocs/parameters.py:21]: drf serializers map to query parameters

File: src/sentry/apidocs/parameters.py [hunk 2]
```diff
@@ -46,6 +91,51 @@ class ISSUE_ALERT_PARAMS:
     )
 
 
+class VISIBILITY_PARAMS:
+    QUERY = OpenApiParameter(
```
Comment (AniketDas-Tekky) [inline src/sentry/apidocs/parameters.py:95]: Is this a visibility-only param? I think we use it in several places outside of visibility.

Comment (wmak) [reply src/sentry/apidocs/parameters.py:95]: This is a specific `Query` to Visibility (see the bit about the search syntax)

**Review by AniketDas-Tekky — APPROVED** [review]

File: src/sentry/apidocs/parameters.py [hunk 3]
```diff
@@ -17,6 +18,50 @@ class GLOBAL_PARAMS:
...
+- `w` for weeks
+
+For example `24h`, to mean query data starting from 24 hours ago to now.""",
+    )
+    START = OpenApiParameter(
+        name="start",
+        location="query",
+        required=False,
+        type=OpenApiTypes.DATETIME,
+        description="The start of the period of time for the query, expected in ISO-8601 format. For example 2001-12-34T12:34:56.7890",
```
Comment (edwardgou-sentry) [inline src/sentry/apidocs/parameters.py:40]:
```suggestion
        description="The start of the period of time for the query, expected in ISO-8601 format. For example `2001-12-14T12:34:56.7890`",
```
I think `2001-12-34T12:34:56.7890` might be an invalid date due to 34th day 😆

File: src/sentry/apidocs/parameters.py [hunk 4]
```diff
@@ -17,6 +18,50 @@ class GLOBAL_PARAMS:
...
+    )
+    END = OpenApiParameter(
+        name="end",
+        location="query",
+        required=False,
+        type=OpenApiTypes.DATETIME,
+        description="The end of the period of time for the query, expected in ISO-8601 format. For example 2001-12-34T12:34:56.7890",
```
Comment (edwardgou-sentry) [inline src/sentry/apidocs/parameters.py:47]:
```suggestion
        description="The end of the period of time for the query, expected in ISO-8601 format. For example `2001-12-14T12:34:56.7890`",
```

File: src/sentry/apidocs/parameters.py [hunk 5]
```diff
@@ -17,6 +18,50 @@ class GLOBAL_PARAMS:
...
+    )
+    PROJECT = OpenApiParameter(
+        name="project",
+        location="query",
+        required=False,
+        many=True,
+        type=int,
+        description="The ids of projects to filter by. `-1` means all available projects, if parameter omitted means 'My Projects'",
```
Comment (edwardgou-sentry) [inline src/sentry/apidocs/parameters.py:55]:
```suggestion
        description="The ids of projects to filter by. `-1` means all available projects. If this parameter is omitted, the request will default to using 'My Projects'",
```

**Review by edwardgou-sentry — COMMENTED** [review]
Have some suggestions! Feel free to pick and choose or edit.

**Comment by edwardgou-sentry** [comment]
Also, probably out of scope for this pr, but I think this part should say something like, ```You need authorization via bearer auth token```, since authentication is a different thing. This part lives in the docs repo though so definitely not for this pr.

image

File: src/sentry/api/endpoints/organization_events.py [hunk 6]
```diff
@@ -127,10 +133,77 @@ def data_fn(offset, limit):
...
+                            "transaction": "string",
+                        },
+                    },
+                },
+            )
+        ],
+    )
     def get(self, request: Request, organization) -> Response:
+        """
+        Retrieves discover (aka. events) data for a given organization.
```
[the remark this answers is not in the document]
Comment (wmak) [reply src/sentry/api/endpoints/organization_events.py:198]: hmm thinking about it probably should avoid both shorthands, going to go with the full:

```suggestion
        Retrieves discover (also known as events) data for a given organization.
```

**Pull request merged**